src/ocean/util/app/ext/LogExt.d

```diff
@@ -152,18 +152,27 @@ class LogExt : IConfigExtExtension
         auto log_config = ConfigFiller.iterate!(LogUtil.Config)("LOG", config);
         auto log_meta_config = ConfigFiller.fill!(LogUtil.MetaConfig)("LOG", config);
 
+        enable_loose_parsing(conf_ext.loose_config_parsing);
```

Comment:
Any reason to move this ?

Reply:
Yep, I have spent some minutes thinking why does it matter to call this right before logger configuration, realized it doesn't matter and moved it to the top to prevent further confusion :angel:

Reply:
I'm fine with the move, I find it clearer also, was just wondering if there was an impact on the code I was not seeing.